src/clr-addons/paged-search-result-list/paged-search-result-list.ts

```diff
@@ -34,7 +34,6 @@ export class ClrPagedSearchResultList {
   /**
    * The template how each item should be displayed.
    */
-  @Input('clrItemTemplate')
```

Comment:
Simply removing the Input might break compatibility with applications. But as It is not even documented, that one can use a clrItemTemplate input, it might be a good idea to remove it.

Reply:
I also don't know any user of this input field, so I want to let it die and only revert it in a patch if someone really needs it